instrumentation-security/spring-webmvc-5.3.0/src/main/java/org/springframework/web/servlet/handler530/SpringHelper.java

```diff
@@ -0,0 +1,35 @@
+package org.springframework.web.servlet.handler530;
+
+import com.newrelic.api.agent.security.NewRelicSecurity;
+import com.newrelic.api.agent.security.schema.ApplicationURLMapping;
+import com.newrelic.api.agent.security.instrumentation.helpers.*;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.servlet.mvc.condition.PathPatternsRequestCondition;
+import org.springframework.web.servlet.mvc.condition.PatternsRequestCondition;
+import org.springframework.web.servlet.mvc.method.RequestMappingInfo;
+import org.springframework.web.util.pattern.PathPattern;
+
+import java.util.Iterator;
+
+public class SpringHelper {
+    public static <T> void gatherURLMappings(T mapping){
+        try {
+            RequestMappingInfo mappingInfo = (RequestMappingInfo) mapping;
+            PatternsRequestCondition patternsCondition = mappingInfo.getPatternsCondition();
+            PathPatternsRequestCondition pathPatternsCondition = mappingInfo.getPathPatternsCondition();
+            for (RequestMethod method : mappingInfo.getMethodsCondition().getMethods()) {
+                if (patternsCondition != null)
+                    for (String url : patternsCondition.getPatterns()) {
+                        URLMappingsHelper.addApplicationURLMapping(new ApplicationURLMapping(method.name(), url));
+                    }
+                else if (pathPatternsCondition != null) {
+                    for (PathPattern url : pathPatternsCondition.getPatterns()) {
+                        if (url != null)
```

Comment:
missing curly braces, proper formatting.

Reply:
Fixed this in commit [2875fd0](https://github.com/newrelic/csec-java-agent/pull/53/commits/2875fd03e64b3ffa9a2115244c19655c0c82bf0b).